public/style/main.css

```diff
@@ -128,6 +164,127 @@ nav div.section {
   overflow-y       : hidden;
   overflow-x       : hidden;
 }
+
+/* BOTTOM-UP CONTROL PANEL MODULES */
+
+.controlModules {
+  font-size: 1.2em;
+}
+.table{
+  font-size: .8em;
+}
+
+.iconContainer {
+  border-radius    : 10px;
+  background-color : yellow;
+  border-color: black;
+  border-width     : 1px;
+  color            : black;
+  margin-bottom: 0px;
+  opacity: 1;
+  padding: 4px;
+
+}
+
+.destinationIcon {
+  font-size: 1.2em;
+}
+
+.caption{
+  font-size: .6em;
+}
+
+
+.basecol_POI {
+  position         : absolute;
+  min-height: 23%;
```

Comment:
this brings up why we should be nesting divs because then we could also nest classes as a result